### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by anonymous-penguin** [devongovett/atom-jade#41]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (C:\Users\**********\AppData\Local\atom\app-0.190.0\resources\app\src\package.js:550:9)
<unknown> (C:\Users\**********\AppData\Local\atom\app-0.190.0\resources\app\src\package.js:172:43)
```

**Comment by LinusU** [comment]
Dupe of #30

**Comment by LinusU** [comment]
Fixed in 0.4.0
